content/en/features/disaster-recovery.md

```diff
@@ -27,28 +27,32 @@ plgd Gateways are subscribed to the EventBus to notify you through the gRPC stre
 An active subscription to plgd events, using gRPC, WebSockets, NATS, or JetStream might fail. Same as the publish operation executed by the plgd service. There are multiple reasons which we cannot prevent - infrastructure failure when the node went down, service crash, or operator failure during the roll-out. To be able to get your system back to the in-sync state, failure has to be detected and the data reconciliation process started.
 
 ### Missing Events Detection
+
 Sometimes it might be obvious that some events might be lost. If your service consuming plgd events restarted due to a crash, network outage, or infrastructure failure, there is a high chance that you missed some events. In such a scenario, the service shall start right away it's up and running with the data reconciliation process.
 
 Another, not so obvious event data loss might occur due to EventBus service disruptions or during a very short network outage. Your service wasn't restarted; your messaging client just missed one event.
 If your service is subscribed to events and requires the processing of all events in the correct order - skipping one event is not accepted, your service needs to track the version of each [event](https://github.com/plgd-dev/cloud/blob/v2/resource-aggregate/pb/events.proto). In case the **received event's version isn't incremented by one compared to your latest event**, your service shall start the resource reconciliation process.
 
 ### Data Reconciliation using gRPC
+
 #### Interest in the latest resource content only
-The plgd gRPC Gateway exposes an RPC call(TODO) to retrieve the latest version of the resource. In case you are not interested in all the changes which occurred during your outage, this is the right way how to get in sync. Additionally, to optimize this operation, timestamp ETag can be specified. Just take the latest event you have persisted in and pass the `timestamp_ms` from the `EventMetadata` as an argument. If no update of that resource occurred after the specified time, you won't receive any resource data.
+
+The plgd gRPC Gateway exposes `GetEvents` RPC call to retrieve the latest version of the resource. In case you are not interested in all the changes which occurred during your outage, this is the right way how to get in sync. Additionally, to optimize this operation, timestamp ETag can be specified. Just take the latest event you have persisted in and pass the `timestamp_ns` from the `EventMetadata` as an argument. If no update of that resource occurred after the specified time, you won't receive any resource data.
```

Comment:
`timestamp_ns` was replaced by `timestamp` https://github.com/plgd-dev/cloud/blob/v2/resource-aggregate/pb/events.proto#L12 - pls add the note about `ns`